src/riak_kv_qry_buffers.erl

```diff
@@ -688,11 +822,14 @@ touch_qbuf(QBufRef, State0 = #state{qbufs = QBufs0}) ->
     State0#state{qbufs = lists:keyreplace(
                            QBufRef, 1, QBufs0, {QBufRef, QBuf9})}.
 
-kill_ldb(RootPath, Table, LdbRef) ->
+kill_ldb(RootPath, Table, LdbRef) when LdbRef /= undefined ->
     ok = riak_kv_qry_buffers_ldb:delete_table(Table, LdbRef, RootPath),
+    ok;
+kill_ldb(_RootPath, _Table, _DelayedAndNeverCalledFun) ->
```

Comment:
This function will be gone in https://github.com/basho/riak_kv/pull/1601, so I'll spare a commit. Will take note of the guard style guide item.